packages/devtools/lib/src/inspector/inspector_tree_canvas.dart

```diff
@@ -172,6 +172,8 @@ class InspectorTreeNodeCanvasRender
   // TODO(jacobr): share this method with test code.
   @override
   Icon hitTest(Offset location) {
+    if (offset == null) return null;
```

Comment:
slightly related edge case that hit while debugging a broken version of this cl.